app/NewApp.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Suyabay;
+
+use Illuminate\Database\Eloquent\Model;
+
+class NewApp extends Model
```

Comment:
**Code Quality - Critical**
@andela-araimi This is a bad Model name, please change this.